contracts/strategies/QVImpactStream.sol

```diff
@@ -0,0 +1,205 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.19;
+
+// External Libraries
+import {Multicall} from "openzeppelin-contracts/contracts/utils/Multicall.sol";
```

Comment:
Why do we need this library for? I think if we have batch functions we dont need it.